contracts/story-nft/BaseStoryNFT.sol

```diff
@@ -17,72 +16,54 @@ import { IStoryNFT } from "../interfaces/story-nft/IStoryNFT.sol";
 ///         To create a new custom StoryNFT, inherit from this contract and override the required functions.
 ///         Note: the new StoryNFT must be whitelisted in `StoryNFTFactory` by the Story governance in order
 ///         to use the Story NFT Factory features.
-abstract contract BaseStoryNFT is IStoryNFT, ERC721URIStorage, Ownable, Initializable {
+abstract contract BaseStoryNFT is IStoryNFT, ERC721URIStorageUpgradeable, OwnableUpgradeable {
     /// @notice Story Proof-of-Creativity IP Asset Registry address.
     IIPAssetRegistry public immutable IP_ASSET_REGISTRY;
 
     /// @notice Story Proof-of-Creativity Licensing Module address.
     ILicensingModule public immutable LICENSING_MODULE;
```

Comment:
Same as above.
add this comments for each immutable var.
/// @custom:oz-upgrades-unsafe-allow state-variable-immutable

Reply:
resolved in [d737282](https://github.com/storyprotocol/protocol-periphery-v1/pull/109/commits/d737282ad16bd860d8253fd0475dbd31fa68a1d6)